conformance/reports/v1.2.0/kong-gateway-operator/standard-v1.4.0-expressions-report.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: gateway.networking.k8s.io/v1
+date: "2024-11-19T11:51:58+01:00"
+gatewayAPIChannel: standard
+gatewayAPIVersion: v1.2.0
+implementation:
+  contact:
+  - https://github.com/kong/gateway-operator/issues/new/choose
+  organization: Kong
+  project: gateway-operator
+  url: https://github.com/kong/gateway-operator
+  version: v1.4.0
+kind: ConformanceReport
+mode: expressions
+profiles:
+- core:
+    result: partial
+    skippedTests:
+    - HTTPRouteWeight
```

Comment:
Non-blocking, but is this related to https://github.com/kubernetes-sigs/gateway-api/issues/3233?

Reply:
It might be. I lack some context about this specific failure in the Kong project, but yeah it should be a flake, not a consistent failure. So, the possibility this is related to #3233 is high.